src/main/java/eu/xenit/gradle/docker/DockerBuildBehavior.java

```diff
@@ -102,6 +105,12 @@ public void execute(Project project) {
 
         Task task = project.getTasks().getAt("composeUp");
         task.dependsOn(buildDockerImage);
+
+        DockerRemoveImage cleanDockerImage = project.getTasks().create("cleanDockerImage", DockerRemoveImage.class);
+        cleanDockerImage.setGroup(GRADLE_DOCKER_TASK_GROUP);
+        cleanDockerImage.targetImageId(buildDockerImage.getImageId());
+        cleanDockerImage.getForce().set(true);
+        cleanDockerImage.mustRunAfter(buildDockerImage);
```

Comment:
so what happens when `buildDockerImage` is NOT invoked ? Like `./gradlew cleanDockerImage` ?

Reply:
You should get an error. Task will fail.